src/Fmt_ast.ml

```diff
@@ -621,6 +298,8 @@ let fmt_variance = function
   | Contravariant -> fmt "-"
   | Invariant -> fmt ""
 
+let fmt_private_flag flag = fmt_if Poly.(flag = Private) "@ private"
```

Comment:
Yes but this function didn't need to be in the big `let rec ... and ... ` recursive definition so I put it outside.

Reply:
Yeah, I know, but that didn't seem to be a big deal (it's clear from the definition of this little function that it can't cause divergence). But at the beginning I was referring to parsetree.mli all the time, which is less true now.